ssl/statem/statem_lib.c

```diff
@@ -788,6 +788,44 @@ static int ssl_method_error(const SSL *s, const SSL_METHOD *method)
 }
 
 /*
+ * ssl_strict_version_check - Check that the specified `version` is supported by
+ * `SSL *` instance
+ *
+ * @s: The SSL handle for the candidate method
+ * @version: Protocol version to test against
+ *
+ * Returns 1 on success, otherwise 0
+ */
+int ssl_strict_version_check(const SSL *s, int version)
```

Comment:
Could you maybe rename this to ssl_version_supported()?

And change the comment to "Returns 1 when supported"